README.md

```diff
@@ -1,49 +1,24 @@
 ## Generating the static Apache ZooKeeper website
 
-In this directory you will find text files formatted using Markdown, with an `.md` suffix.
+In the `src/main/resources/markdown` directory you will find text files formatted using Markdown, with an `.md` suffix.
 
-Building the site requires [Jekyll](http://jekyllrb.com/docs) 3.6.2 or newer. 
-The easiest way to install jekyll is via a Ruby Gem. Jekyll will create a directory called `_site` 
-containing `index.html` as well as the rest of the compiled directories and files. _site should not
-be committed to git as this is the generated content.
-
-To install Jekyll and its required dependencies, execute `sudo gem install jekyll pygments.rb` 
-and `sudo pip install Pygments`. See the Jekyll installation page for more details.
+Building the site requires [Maven](http://maven.apache.org/) 3.5.0 or newer. 
+The easiest way to [install Maven](http://maven.apache.org/install.html) depends on your OS.
+The build process will create a directory called `target/html` containing `index.html` as well as the rest of the
+compiled directories and files. `target` should not be committed to git as it is generated content.
 
 You can generate the static ZooKeeper website by running:
 
-1. `jekyll build` in this directory.
-2. `cp -RP _released_docs _site/doc` - this will include the documentation (see "sub-dir" section below) in the generated site.
+1. `mvn clean install` in this directory.
+2. `cp -RP _released_docs _target/html` - this will include the documentation (see "sub-dir" section below) in the generated site.
```

Comment:
After 'mvn clean install' there is no _target dir but only target, is this a typo or am I missing any step?

Reply:
Thanks for noting it, it was a typo. Fixed.

Reply:
Hi @lvfangmin, thanks for the detailed review.

Yes it is the right page. The documentation links point into the void, since the documentations are generated in a separate process and are manually copied into their final place. Once the generated page is copied into its place the links won't be broken.

The wiki page must be updated after the content will be changed. I've already got my account, so am able to edit it, just wait for the commit of this change.

Reply:
Thanks @tamaashu, can you update the readme to point out how to copy those doc? 

One suggest, the top and bottom layout is a bit strange, they're too wide comparing to the main content on that page, can we make those not that wide?